tests/stress.rs

```diff
@@ -4753,3 +4734,22 @@ pub unsafe extern "C" fn stress_functions(context: &dc_context_t) {
         dc_lot_unref(res);
     };
 }
+
+unsafe extern "C" fn dummy_event_handler(
+    _context: &dc_context_t,
+    _event: Event,
+    _data1: uintptr_t,
+    _data2: uintptr_t,
+) -> uintptr_t {
+    0
+}
+
+#[test]
+fn run_stress_tests() {
+    let context = dc_context_new(
+        dummy_event_handler,
```

Comment:
@dignifiedquire Not sure how to pass a null function as event handler so added dummy for now.